packages/cw1155/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+pub use crate::msg::{Cw1155HandleMsg, TokenId};
+pub use crate::query::{
+    ApprovedForAllResponse, BalanceResponse, BatchBalanceResponse, Cw1155QueryMsg,
+};
+pub use crate::receiver::{Cw1155BatchReceiveMsg, Cw1155ReceiveMsg};
+
+mod msg;
+mod query;
+mod receiver;
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn it_works() {
+        assert_eq!(2 + 2, 4);
+    }
+}
```

Comment:
```suggestion
```

Reply:
You need at least one test so `cargo test` doesn't error on this package.

Good to remove this as soon as you write any reasonable test of the real code.